README.md

```diff
@@ -1,4 +1,28 @@
-## Etherscan API consumption in Golang
+# Etherscan API consumption in Golang
 
-clone the repo: `git clone https://github.com/Amenokal-Labs/mev-mantis.git`
-navigate to the cloned folder, then run: `go run extract.go`
\ No newline at end of file
+## Instructions to run
+
+clone the repo: `git clone https://github.com/Amenokal-Labs/mev-mantis.git`.
+
+navigate to the cloned folder, then run: `go run extract.go`.
+
+## Docs
+<<<<<<< HEAD
```

Comment:
please remove lines starting with `<<<<<<<` and `>>>>>>>`. they come from the rebase process